fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.js

```diff
@@ -112,123 +117,183 @@ function ProductDetail(props) {
-                    {props.inStock && props.availableQuantity && props.availableQuantity > 0 &&
-                        <div className="product-detail__in-stock">
-                            {props.inStockLabel} {props.availableQuantity}
-                            {props.expectedDelivery && 
-                                <div className="product-detail__expected-delivery">{props.expectedDeliveryLabel} {moment.utc(props.expectedDelivery).local().format("L")}</div>
-                            }
-                        </div>
-                    }
-                    {props.inOutlet && props.availableOutletQuantity && props.availableOutletQuantity > 0 &&
-                        <div className="product-detail__in-stock">
-                            {props.inOutletLabel} {props.availableOutletQuantity}
+            <div className="product-detail__container">
+                <div className="product-detail__head columns is-desktop">
+                    <div className="product-detail__gallery-column">
+                        <div className="column desktop-product-gallery">
+                            <div className="is-flex is-flex-wrap product-detail__product-gallery">
+                                {props.images && props.images.length > 0 && images.map((image, index) => {
+                                    return (
+                                        <div className="product-detail__gallery-column__desktop-product-image" key={index}>
+                                            <LazyLoad offset={LazyLoadConstants.defaultOffset()}>
+                                                <ResponsiveImage sources={image.sources} imageSrc={image.src} imageAlt={image.alt} />
+                                            </LazyLoad>
+                                        </div>
+                                    )
+                                })}
+                                {props.images.length > 6 &&
+                                    <div className="product-detail__gallery-column__desktop-more-product-images">
+                                        {showMoreImg ? (
+                                            <div className="is-flex is-justify-content-center">                                                                                                
+                                                <button variant="outlined" className="product-detail__gallery-column__desktop-more-product-images__button-more-less" onClick={handleShowMoreImg}>Zobacz mniej</button>
+                                            </div>
+                                        ) : (
+                                            <div className="is-flex is-justify-content-center">
+                                                <button variant="outlined" className="product-detail__gallery-column__desktop-more-product-images__button-more-less" onClick={handleShowMoreImg}>Zobacz więcej</button>                                                
```

Comment:
**suggestion:** Use Button component from MUI library and make more readable code